frontend/test/pytest/test_measurements_results.py

```diff
@@ -668,5 +668,327 @@ def expected(x, measurement):
         assert np.allclose(result[5], expected(x, qml.state()))
 
 
+class TestNewArithmeticOps:
+    "Test PennyLane new arithmetic operators"
+
+    @pytest.mark.parametrize(
+        "meas, expected",
+        [[qml.expval, np.array(-0.70710678)], [qml.var, np.array(0.5)]],
+    )
+    def test_prod_xzi(self, meas, expected, backend):
+        """Test ``qml.ops.op_math.Prod`` converting to TensorObs."""
+
+        @qjit
+        @qml.qnode(qml.device(backend, wires=3))
+        def circuit(x: float, y: float):
+            qml.RX(x, wires=0)
+            qml.RX(y, wires=1)
+            qml.RX(x + y, wires=2)
+            qml.CNOT(wires=[0, 1])
+            return meas(
+                qml.ops.op_math.Prod(
+                    qml.PauliX(wires=0), qml.PauliZ(wires=1), qml.Identity(wires=2)
+                )
+            )
+
+        result = circuit(np.pi / 4, np.pi / 2)
+        assert np.allclose(expected, result)
+
+        circuit(0, 1)  # another call
```

Comment:
What is the intention here?

Reply:
Nothing! there was an issue somewhere else that I thought it came from re-calling the circuit... removed  [8ce5458](https://github.com/PennyLaneAI/catalyst/pull/250/commits/8ce54582d0f8958e7d95979cb3c5fbbbb272aff8) Thanks!